tasks/system_probe.py

```diff
@@ -38,17 +39,28 @@ def build(ctx, race=False, incremental_build=False):
         "BuildDate": datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S"),
     }
 
+    gobin = 'go'
```

Comment:
There's a bit of duplication here, wondering if you maybe would prefer a utility function to get the `gobin`?